code/game/machinery/adv_med.dm

```diff
@@ -327,11 +327,11 @@
 			immune.attack(occupant,usr)
 
 
-/obj/machinery/bodyscanner/proc/get_occupant_data()
-	if (!occupant || !istype(occupant, /mob/living/carbon/human))
-		return
-	var/mob/living/carbon/human/H = occupant
-	var/list/occupant_data = list(
+/proc/get_occupant_data(mob/living/M)
+/*	if (!occupant || !istype(occupant, /mob/living/carbon/human))
```

Comment:
why is this commented instead of deleted?

also need sanity elsewhere since you're removing it here

Reply:
the reason I commented it our was to leave a mark in case of consequence
sanity check however still exists in other functions as proc is called through with several cnecks already

Reply:
Go ahead and take it out if your testing has shown you're covered elsewhere